mypy/checkmember.py

```diff
@@ -17,6 +17,8 @@
 from mypy.semanal import self_type
 from mypy import messages
 from mypy import subtypes
+if False:  # import for forward declaration only
```

Comment:
I tried that first, actually, but it doesn't work.  After removing this conditional import, using the type `'mypy.checker.TypeChecker'` results in `error: Name 'mypy' is not defined`.  (And if I add `import mypy` I get `Name 'mypy.checker.TypeChecker' is not defined`.)